docs/content/docs/get-started/introduction.md

```diff
@@ -23,3 +23,98 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
+
+# Welcome to Flink CDC 🎉
+
+Flink CDC is a stream data integration framework that aims to provide users with
+a more robust API. It allows users to configure their data synchronization logic
+through customized Flink operators and job submission tools. The framework
```

Comment:
It allows users to describe their ETL pipeline logic via YAML elegantly and help users automatically generating customized Flink operators and submitting job.